### ISSUE TITLE
jwt.encode() returns a str which does not have a decode method.

### ISSUE THREAD

**Issue by alexzapata** [PureStorage-OpenConnect/py-pure-client#6]
https://github.com/PureStorage-OpenConnect/py-pure-client/blob/589b2c0dc0a5e890eb74902a8056e43723400d50/pypureclient/token_manager.py#L64

Sorry, I'm not a python guy, but I was doing some experimenting with this and noticed that this errors because jwt.encode() returns a str. If you have line 64 return new_jwt by itself, this should fix it.

**Comment by sdodsley** [comment]
Yes - I agree - I have raised this internally within Pure to try and get this resolved.

**Comment by sdodsley** [comment]
There is a typo in the pip installation `setup.py` that is now creating an upper constraint for PyJWT. 
Until we get this resolved, downgrade your PyJWT to 1.7.1
